config/services/open-balena-vpn.service

```diff
@@ -1,42 +0,0 @@
-# Copyright (C) 2015 Balena Ltd.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as published
-# by the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-
-[Unit]
-Description=open-balena-vpn
-Requires=confd.service haproxy.service
-After=confd.service
-StartLimitIntervalSec=0
-
-[Service]
-StandardOutput=journal+console
-StandardError=journal+console
-WorkingDirectory=/usr/src/app
-EnvironmentFile=/usr/src/app/config/env
-ExecStart=/usr/src/app/bin/start.sh
-ExecStop=-/usr/src/app/bin/stop.sh
-# https://www.freedesktop.org/software/systemd/man/systemd.kill.html#KillMode=
-# only send SIGTERM to master process, send SIGKILL to all remaining processes in the
-# group after TimeoutStopSec has elapsed; this setting prevents VPN worker(s) from
-# restarting after SIGTERM is received and before SIGKILL
-KillMode=mixed
```

Comment:
`KillMode=mixed` has been replaced by shell signal traps, and the SIGINT is forwarded to the main process (in some cases after calling the ExecStop script first).